### PR TITLE
Unify zipkin address in helm

### LINKED ISSUE #11334: Zipkin address inconsistently used through Helm charts
**Describe the bug**
Some Helm charts currently in `master` make use of a `.Values.global.tracer.zipkin.address` value to point to an external Zipkin instance but this doesn't happen in all the charts.

**Expected behavior**
If the external Jaeger/Zipkin is specified, the address should be used consistently throughout.

**Steps to reproduce the bug**
A correct implementation is seen in the [`gateways/templates/deployent.yaml`](https://github.com/istio/istio/blob/b8e30e07ec7b36cf1ba1b445b78c81fdf0d417b9/install/kubernetes/helm/subcharts/gateways/templates/deployment.yaml#L113) where it starts with an explicitly specified Zipkin endpoint, falls back to a namespace-qualified endpoint, and finally ends with a default endpoint.

- [`ingress/templates/deployment.yaml`](https://github.com/istio/istio/blob/3b8093d1635f9ccab4114d5bca3cfdfe7c6968ee/install/kubernetes/helm/subcharts/ingress/templates/deployment.yaml#L55) uses only the fixed default endpoint.
- [`mixer/templates/deployment.yaml`](https://github.com/istio/istio/blob/3b8093d1635f9ccab4114d5bca3cfdfe7c6968ee/install/kubernetes/helm/subcharts/mixer/templates/deployment.yaml#L46) uses the specific endpoint or the default, but not the namespace-qualified endpoint.
- [`templates/configmap.yaml`](https://github.com/istio/istio/blob/3b8093d1635f9ccab4114d5bca3cfdfe7c6968ee/install/kubernetes/helm/istio/templates/configmap.yaml#L142) uses the specific endpoint and the release namespace qualified endpoint, but no default and ignores the value for `global.istioNamespace`.

**Version**
Whatever's in `master` as of January 28, 2019. I'm trying to get a separate Jaeger install working in 1.0.5 so I'm manually grabbing things like this out of `master` and finding it's not easy to get it all lined up with the various inconsistencies.

### PR DESCRIPTION
**Pull request by bianpengyuan**
fix #11334

### CODE REVIEW

File: install/kubernetes/helm/istio/templates/configmap.yaml [hunk 1]
```diff
@@ -143,8 +143,10 @@ data:
           # Address of the Zipkin collector
         {{- if .Values.global.tracer.zipkin.address }}
           address: {{ .Values.global.tracer.zipkin.address }}
+        {{- else if .Values.global.istioNamespace }}
```
Comment (morvencao) [inline install/kubernetes/helm/istio/templates/configmap.yaml]: Why not change `.Values.global.istioNamespace` to `{{ .Release.Namespace }}`?
Helm/Tiller are supposed to manage resource in same namespace.

Comment (bianpengyuan) [reply install/kubernetes/helm/istio/templates/configmap.yaml]: This is to construct an arg for tracing address, but not a resource. But anyway if it is general practice to use release.namespace, I think it also makes sense to use it as default domain here.

File: install/kubernetes/helm/subcharts/mixer/templates/deployment.yaml [hunk 2]
```diff
@@ -183,6 +183,8 @@
           {{- end }}
           {{- if $.Values.global.tracer.zipkin.address }}
           - --trace_zipkin_url=http://{{- $.Values.global.tracer.zipkin.address }}/api/v1/spans
+          {{- else if $.Values.global.istioNamespace }}
```
Comment (morvencao) [inline install/kubernetes/helm/subcharts/mixer/templates/deployment.yaml]: Also here :)

Comment (bianpengyuan) [reply install/kubernetes/helm/subcharts/mixer/templates/deployment.yaml]: Done.

**Pull request merged**